docs/digger/installation/macos-options.asciidoc

```diff
@@ -0,0 +1,229 @@
+the Ansible job.
+
+.Install Xcode variables
+|===
+| Variable name | Description
+
+|xcode_install_user
+|Apple Developer Account username. If this is not set then Xcode will not be
+installed.
+|xcode_install_password
+|Apple Developer Account password. If this is not set then Xcode will not be
+installed.
+|xcode_install_session_token
+|Apple Developer Account auth cookie from fastlane spaceauth command (For
+accounts with 2FA enabled).
+|xcode_versions
+|A list of Xcode versions to install. These may take over 30 minutes each to
+install.
+|===
+
+== Download Apple certificates
+
+Downloads some required certificates into the node. Right now, the only
+required certificate is Apple's WWDR certificate. This certificate will be
+downloaded into the user's home directory.
+
+To run this section as a standalone step you must specify the
+`osx_download_certs` tag.
+
+.Download Apple certificate variables
+|===
+| Variable name | Description
+
+|apple_wwdr_cert_url
+|Apple WWDR certificate URL. Defaults to Apple's official URL
+|apple_wwdr_cert_file_name
+|Output file name of the downloaded file. Defaults to AppleWWDRCA.cer.
+|===
+
+== Update Cocoapods
+
+Executes pod repo update.
+
+To run this section as a standalone step you must specify the
+`osx_pod_repo_update` tag.
+
+== Configure Buildfarm node
+
+Creates a credential set in the build farm for the macOS nodes using the
+provided keys. Add each machine as a node in the build farm, connecting through
+SSH.
+
+You will need to create a key pair using a tool such as ssh-keygen to allow the
+Jenkins instance to connect with the macOS nodes.
+
+To run this section as a standalone step you must specify the
+`osx_configure_buildfarm` tag.
+
+.Configure buildfarm node variables
+|===
+| Variable name | Description
+
+|credential_private_key_path
+|Location of the private key of the pair. This is stored in Jenkins and used to
+SSH into the macOS node. If this is not set then this section will be skipped.
+|credential_passphrase
+|Passphrase of the private key. This is stored in Jenkins and used to SSH into
+the macOS node. If this is not set then this section will be skipped.
+|credential_public_key_path
+|Location of the public key of the pair. If this is not set then this section
+will be skipped.
+|buildfarm_node_port
+|The port to connect to the macOS node on. Defaults to 22.
+|buildfarm_node_root_dir
+|Root node of the node in Jenkins. Defaults to /Users/jenkins.
+|buildfarm_credential_id
+|Identifier for the Jenkins credential object. Defaults to
+macOS_buildfarm_cred.
+|buildfarm_credential_description
+|Description of the Jenkins credential object.
+|buildfarm_node_name
+|Name of the slave/node in Jenkins. Defaults to macOS (<node_host_address>).
+|buildfarm_node_labels
+|List of labels to give the macOS node. Defaults to only ios.
+|buildfarm_user_id
+|Jenkins user. Defaults to admin.
+|buildfarm_node_executors
+|Number of executors (Jenkins configuration) on the macOS node. Defaults to 1.
```

Comment:
I think we need to warn user here it is not a good idea to change this value. Mainly because each build may use a different signing credential, and if they allow concurrent builds on ios, there will be no guarantee that the right signing credential will be used.

So unless they are sure all the apps will be built using the same credentials, this value should not be changed